app/screens/route-list/route-list-screen.tsx

```diff
@@ -120,7 +120,7 @@ export const RouteListScreen = observer(function RouteListScreen({ navigation, r
           onRefresh={() => trains.refetch()}
           refreshing={trains.isRefetching}
           renderItem={renderRouteCard}
-          keyExtractor={(item) => item.trains[0]?.trainNumber.toString()}
+          keyExtractor={(item) => item.trains.map((train) => train.trainNumber).join()}
```

Comment:
Why this change was needed? 
Are there scenarios where the same train is being used for 2 different routes?

Reply:
Yes, at one time the API gave me 2 routes with a same first train. Probably just a glitch at the Rail API but I think we need to support it (react threw me a duplicate key error)

Reply:
Interesting. Mind sharing what the route was?

Reply:
Ben Gurion Airport to Be'er Sheva Center on 4/4/23 at 9:44.

Here's the API response:
<img width="527" alt="image" src="https://user-images.githubusercontent.com/12946462/230374088-71ff3c38-62b6-46a0-97f8-1878909380ca.png">